### DISCUSSION TITLE
Release process

### DISCUSSION THREAD

**Discussion by jvancoillie** [Guikingone/SchedulerBundle#50] [Q&A] [answered]
Hello dear @Guikingone ,

Do you have a time-based publishing process model ?
Or simply an approximate delivery date for version 0.4.0 that I can adapt my development accordingly

**Comment by Guikingone** [comment, marked as answer]
Hi @jvancoillie 👋🏻 

The actual "release process" is the following: 

- Bug fixs released ASAP when the API is not impacted
- A minor version every 20-25 days depending on the evolutions / bug fixs
- The `1.0` version once the API is stable

Regarding the recent bug fixs / improvements, the `0.4` is planned to be released soon (probably by the end of the week), I've planned to ship it with https://github.com/Guikingone/SchedulerBundle/pull/43 but I'm facing issues in the tests so it may take a day or two to make it stable.

Once the `1.0` version is released / tagged, the release process will probably evolve to something like this: 

- Major version every 6 months (depending on the merged API breaks)
- Minor version every months (depending on the merged "non-API breaks")
- Patch version when required and when a bug fix is merged (and if this one does not break the API)
